pycheribuild/config/chericonfig.py

```diff
@@ -292,11 +292,11 @@ def __init__(self, loader, action_class) -> None:
             help="When building with --include-dependencies ignore the SDK dependencies. Saves a lot of time "
                  "when building libc++, etc. with dependencies but the sdk is already up-to-date. "
                  "This is like --no-include-toolchain-depedencies but also skips the target that builds the sysroot.")
-        self.skip_dependency_filters = loader.add_option(
+        self.skip_dependency_filters: typing.List[re.Pattern] = loader.add_option(
```

Comment:
```suggestion
        self.skip_dependency_filters: "list[re.Pattern]" = loader.add_option(
```

I've been trying to remove typing.List imports, so I'd suggest using the string version.